commit f0665940e7b411ebb81e64fbbf88dbaa3fd57582
Author: bertin <macportsuser@macports.org>
Date:   Thu May 22 23:55:20 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-no-voodoo.diff

diff --git a/Mk/macports.tea.mk b/Mk/macports.tea.mk
index ce20a9f31..18c87e9e4 100644
--- a/Mk/macports.tea.mk
+++ b/Mk/macports.tea.mk
@@ -9,7 +9,7 @@ $(SHLIB_NAME): ${OBJS}
 	${SHLIB_LD} ${OBJS} -o ${SHLIB_NAME} ${TCL_STUB_LIB_SPEC} ${SHLIB_LDFLAGS} ${LIBS}
 
 pkgIndex.tcl: $(SHLIB_NAME)
-	$(SILENT) ../pkg_mkindex.sh . || ( rm -rf $@ && exit 1 )
+	../pkg_mkindex.sh . || ( rm -rf $@ && exit 1 )
 
 clean::
 	rm -f ${OBJS} ${SHLIB_NAME} so_locations pkgIndex.tcl
